src/subdomains/voting/api/voting.controller.ts

```diff
@@ -38,6 +38,12 @@ export class VotingController {
     return this.votingService.result;
   }
 
+  @Get('user-result')
+  @ApiOkResponse({ type: CfpResultDto, isArray: true })
+  async getCurrentUserResult(): Promise<CfpUserInfo[]> {
+    return this.votingService.getVoteDetails();
+  }
+
```

Comment:
Or maybe we should just name the route `result/votes` and the DTO just `CfpVoteDto`? It's even not related to a user, only to a deposit address 😆